etl/steps/data/garden/demography/2023-03-31/population/meta.yml

```diff
@@ -0,0 +1,80 @@
+dataset:
+  namespace: demography
+  short_name: population
+  title: Population (various sources, 2023.1)
+  description: |
+    This dataset has been created using multiple sources:
+
+    • UN WPP (2022): Covers the period 1950-2100. Future projections are based on the Medium variant.
+    • Gapminder (v7): Mostly covers the period 1800-1949. In addition, it complements the dataset with population data for the "Vatican" until 2100 (it is missing in UN WPP's estimates).
+    • HYDE (v3.2): Mostly covers the period 10,000 BCE - 1799. In addition, it complements the dataset with population for
+    some countries outside of this year period. E.g. it provides data for "Svalbard and Jan Mayen" (until 2017) and "Netherlands Antilles" (until 2010).
+    • Gapminder (Systema Globalis): Covers the period 1555-2008. It complements the dataset with former countries and other data points not present in the other sources.
+
+    For a more detailed description, please refer to the field "source" in table "population".
+
+  version: "2022-12-08"
+  sources: &population-sources
+    - name: Gapminder (2022)
+      published_by: Gapminder (v7)
+      url: https://www.gapminder.org/data/documentation/gd003/
+      date_accessed: 2023-03-31
+    - name: UN (2022)
+      published_by: UN, World Population Prospects (2022)
```

Comment:
Again, I'd say `UN` -> `United Nations`.

Reply:
Will keep `UN (2022`. Otherwise, the full citation (`UN (2022), Gapminder (2022); HYDE (2017)`) is too long (even more if combined with other sources).

Reply:
The full citation can be as long as we want, this doesn't show up in charts, but only in the SOURCES tab. But up to you.